notebooks/integrations/hugging-face/loading-model-from-hugging-face.ipynb

```diff
@@ -0,0 +1,1109 @@
+    "        \"type\": \"text\",\n",
+    "        \"fields\": {\n",
+    "          \"keyword\": {\n",
+    "            \"type\": \"keyword\",\n",
+    "            \"ignore_above\": 256\n",
+    "          }\n",
+    "        }\n",
+    "      },\n",
+    "      \"text_embedding\": {\n",
+    "        \"properties\": {\n",
+    "          \"is_truncated\": {\n",
+    "            \"type\": \"boolean\"\n",
+    "          },\n",
+    "          \"model_id\": {\n",
+    "            \"type\": \"text\",\n",
+    "            \"fields\": {\n",
+    "              \"keyword\": {\n",
+    "                \"type\": \"keyword\",\n",
+    "                \"ignore_above\": 256\n",
+    "              }\n",
+    "            }\n",
+    "          },\n",
+    "          \"predicted_value\": {\n",
+    "            \"type\": \"dense_vector\",\n",
+    "            \"dims\": 384,\n",
+    "            \"index\": True,\n",
+    "            \"similarity\": \"l2_norm\"\n",
+    "          }\n",
+    "        }\n",
+    "      }\n",
+    "    }\n",
+    "  }\n",
+    "\n",
+    "INDEX_SETTINGS = {\n",
+    "    \"index\": {\n",
+    "      \"number_of_replicas\": \"1\",\n",
+    "      \"number_of_shards\": \"1\",\n",
+    "      \"default_pipeline\": PIPELINE_ID\n",
+    "    }\n",
+    "}\n",
+    "\n",
+    "if(DELETE_INDEX):\n",
+    "  if es.indices.exists(index=INDEX_NAME):\n",
+    "    print(\"Deleting existing %s\" % INDEX_NAME)\n",
+    "    es.indices.delete(index=INDEX_NAME, ignore=[400, 404])\n",
+    "\n",
+    "print(\"Creating index %s\" % INDEX_NAME)\n",
+    "es.indices.create(index=INDEX_NAME, mappings=INDEX_MAPPING, settings=INDEX_SETTINGS,\n",
+    "                  ignore=[400, 404])\n"
+   ],
+   "metadata": {
+    "id": "xAkc1OVcOxy3",
+    "colab": {
+     "base_uri": "https://localhost:8080/"
+    },
+    "outputId": "b2453634-89b8-48bc-ac65-a6a1c3b8170f"
+   },
+   "execution_count": null,
+   "outputs": [
+    {
+     "output_type": "stream",
+     "name": "stdout",
+     "text": [
+      "Creating index blogs\n"
+     ]
+    },
+    {
+     "output_type": "stream",
+     "name": "stderr",
+     "text": [
+      "<ipython-input-7-8366c643361b>:93: DeprecationWarning: Passing transport options in the API method is deprecated. Use 'Elasticsearch.options()' instead.\n",
+      "  es.indices.create(index=INDEX_NAME, mappings=INDEX_MAPPING, settings=INDEX_SETTINGS,\n"
+     ]
+    },
+    {
+     "output_type": "execute_result",
+     "data": {
+      "text/plain": [
+       "ObjectApiResponse({'acknowledged': True, 'shards_acknowledged': True, 'index': 'blogs'})"
+      ]
+     },
+     "metadata": {},
+     "execution_count": 7
+    }
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "source": [
+    "### Get blogs dataset\n",
+    "Download:\n",
+    "- The example blogs dataset is located in [GitHub](https://github.com/radoondas/flask-elastic-nlp/blob/main/embeddings/blogs/blogs-no-embeddings.json.zip)\n",
```

Comment:
we should update this link to the elasticsearch-labs repo

Reply:
Yes. I forgot to change this link as I already did in the code itself. I'll fix this.